src/Core/Constants.cs

```diff
@@ -170,6 +170,7 @@ public static class FeatureFlagKeys
     public const string SingleTapPasskeyCreation = "single-tap-passkey-creation";
     public const string SingleTapPasskeyAuthentication = "single-tap-passkey-authentication";
     public const string EnableRiskInsightsNotifications = "enable-risk-insights-notifications";
+    public const string EnablePMAuthenticatorSync = "enable-pm-auth-sync";
```

Comment:
🤔 This is fine but in several parts of the mobile projects we're using `bwa` to refer to the Authenticator; would it be better to name the feature flag: `enable-pm-bwa-sync` so it has the two abbreviations of the apps?

Reply:
That is better, I will update it 🙂